go/vm/utils_test.go

```diff
@@ -0,0 +1,43 @@
+package vm
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/holiman/uint256"
+)
+
+func randomUint256(rnd *rand.Rand) *uint256.Int {
+	var value uint256.Int
+	value[0] = rand.Uint64()
+	value[1] = rand.Uint64()
+	value[2] = rand.Uint64()
+	value[3] = rand.Uint64()
+	return &value
+}
+func TestUint256ToValue(t *testing.T) {
+	rnd := rand.New(rand.NewSource(0))
+	value := randomUint256(rnd)
+	result := Uint256ToValue(value)
+	expected := value.Bytes32()
+	if result != expected {
+		t.Errorf("incorrect result, got: %v, want: %v", result, expected)
+	}
+}
+
+func TestUint256ToValue_nil(t *testing.T) {
+	result := Uint256ToValue(nil)
+	expected := [32]byte{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0}
```

Comment:
nit: should be the same as
```suggestion
	expected := [32]byte{}
```